gs_chunked_io/writer.py

```diff
@@ -43,13 +43,11 @@ def __init__(self,
         self._buffer = bytearray()
         self._current_part_number = 0
         self._closed = False
-        if threads is not None:
-            assert 1 <= threads
-            max_workers = max(threads, 8)
-            self.executor: Optional[ThreadPoolExecutor] = ThreadPoolExecutor(max_workers=max_workers)
-            self.future_chunk_uploads = AsyncSet(self.executor, concurrency=threads)
+        self.future_chunk_uploads: Optional[AsyncSet] = None
```

Comment:
I think that this:

```
self.future_chunk_uploads: Optional[AsyncSet] = None
if async_set is not None:
    self.future_chunk_uploads = async_set
else:
    self.future_chunk_uploads = None
```

may be the same as:

```
self.future_chunk_uploads: Optional[AsyncSet] = async_set
```

Reply:
Thanks! will fix.